controllers/istiorevision/istiorevision_controller.go

```diff
@@ -372,6 +400,109 @@ func (r *IstioRevisionReconciler) determineReadyCondition(ctx context.Context,
 	}, nil
 }
 
+func (r *IstioRevisionReconciler) determineInUseCondition(ctx context.Context, rev *v1alpha1.IstioRevision) (v1alpha1.IstioRevisionCondition, error) {
+	isReferenced, err := r.isRevisionReferencedByWorkloads(ctx, rev)
+	if err != nil {
+		return v1alpha1.IstioRevisionCondition{}, err
+	}
+
+	if isReferenced {
+		return v1alpha1.IstioRevisionCondition{
+			Type:    v1alpha1.IstioRevisionConditionTypeInUse,
+			Status:  metav1.ConditionTrue,
+			Reason:  v1alpha1.IstioRevisionConditionReasonReferencedByWorkloads,
+			Message: "The IstioRevision is referenced by at least one pod or namespace",
```

Comment:
Maybe here and below we could simplify these messages by omitting "the istiorevision". It's kind of implicit that we're seeing the status of a istiorevision object. When inspecting the status, I'd completely understand the message if it was just "Referenced by at least one pod or namespace" or "Not referenced by any pod or namespace".

Reply:
+1 
Done